rootfs/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.1
+
+RUN apk add --update-cache curl bash openssl sudo python && rm -rf /var/cache/apk/*
```

Comment:
Just want to double check if you want Python 2.7 or 3.5, it will affect some of the code below